lang/test/org/partiql/lang/eval/EvaluatingCompilerGroupByTest.kt

```diff
@@ -932,4 +953,96 @@ EvaluatingCompilerGroupByTest : EvaluatorTestBase() {
             null)
     }
 
+    @Test
+    fun missingGroupByTest() {
+        checkInputThrowingEvaluationException(
+            "SELECT MAX(@v2), @v2 FROM `[1, 2.0, 3e0, 4, 5d0]` AS v2",
+            session,
+            ErrorCode.EVALUATOR_VARIABLE_NOT_INCLUDED_IN_GROUP_BY,
+            mapOf(
+                    Property.LINE_NUMBER to 1L,
+                    Property.COLUMN_NUMBER to 19L,
+                    Property.BINDING_NAME to "v2"
+            ),
+            null)
+    }
+
+    @Test
+    fun missingGroupBySFWTest() {
+        checkInputThrowingEvaluationException(
+            """
+            SELECT x.title, MIN(money) as price
+            FROM books AS x, x.price AS money
+            WHERE author like '%Milan%'
+            """.trimIndent(),
+            session,
+            ErrorCode.EVALUATOR_VARIABLE_NOT_INCLUDED_IN_GROUP_BY,
+            mapOf(
+                    Property.LINE_NUMBER to 1L,
+                    Property.COLUMN_NUMBER to 8L,
+                    Property.BINDING_NAME to "x"
+            ),
+            null)
```

Comment:
Explicitly passing `null` here isn't required since the parameter has a default value of `null`.

Reply:
Got it. Will remove the unnecessary parameter.